<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateConnection</class>
 <widget class="QDialog" name="CreateConnection">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>370</width>
    <height>419</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>370</width>
    <height>419</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>370</width>
    <height>419</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Create new Database Connection</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/menu/DB_CreateConnection</normaloff>:/menu/DB_CreateConnection</iconset>
  </property>
  <widget class="QLabel" name="labelConnectionName">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>17</y>
     <width>145</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Connection Name:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelDatabaseLocation">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>50</y>
     <width>145</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Database Location:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelDatabaseCriver">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>83</y>
     <width>145</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Database Driver:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelDatabaseServer">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>116</y>
     <width>145</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Database Server:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelDatabaseFile">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>149</y>
     <width>145</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Database File:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelHostname">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>182</y>
     <width>145</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Hostname/IP:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelPort">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>215</y>
     <width>145</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Port:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelUserName">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>248</y>
     <width>145</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>User Name:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelUserToken">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>281</y>
     <width>145</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>User Token:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelPassword">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>314</y>
     <width>145</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Password:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxDatabaseLocation">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>50</y>
     <width>200</width>
     <height>26</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Local</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Network</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="lineEditConnectionName">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>17</y>
     <width>200</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditDatabaseDriver">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>83</y>
     <width>200</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>QSQLITE</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditDatabaseServer">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>116</y>
     <width>200</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>SQLite/SpatialLite</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditDatabaseFile">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>149</y>
     <width>160</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditHostname">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>182</y>
     <width>200</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>not required for SQLite</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditPort">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>215</y>
     <width>200</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>not required for SQLite</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditUserName">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>248</y>
     <width>200</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>not required for SQLite</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditUserToken">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>281</y>
     <width>200</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>vj</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditPassword">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>314</y>
     <width>200</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>not required for SQLite</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonTestConnection">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>350</y>
     <width>200</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>test Connection</string>
   </property>
   <property name="icon">
    <iconset resource="Icons.qrc">
     <normaloff>:/buttons/connect</normaloff>:/buttons/connect</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCancel">
   <property name="geometry">
    <rect>
     <x>267</x>
     <y>383</y>
     <width>93</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
   <property name="icon">
    <iconset resource="Icons.qrc">
     <normaloff>:/buttons/cancel</normaloff>:/buttons/cancel</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonOk">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>383</y>
     <width>95</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
   <property name="icon">
    <iconset resource="Icons.qrc">
     <normaloff>:/buttons/ok</normaloff>:/buttons/ok</iconset>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButtonOpenFile">
   <property name="geometry">
    <rect>
     <x>327</x>
     <y>149</y>
     <width>33</width>
     <height>26</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;open File-Dialog&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset resource="Icons.qrc">
     <normaloff>:/buttons/folder</normaloff>:/buttons/folder</iconset>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEditConnectionName</tabstop>
  <tabstop>comboBoxDatabaseLocation</tabstop>
  <tabstop>lineEditDatabaseDriver</tabstop>
  <tabstop>lineEditDatabaseServer</tabstop>
  <tabstop>lineEditDatabaseFile</tabstop>
  <tabstop>toolButtonOpenFile</tabstop>
  <tabstop>lineEditHostname</tabstop>
  <tabstop>lineEditPort</tabstop>
  <tabstop>lineEditUserName</tabstop>
  <tabstop>lineEditUserToken</tabstop>
  <tabstop>lineEditPassword</tabstop>
  <tabstop>pushButtonTestConnection</tabstop>
  <tabstop>pushButtonOk</tabstop>
  <tabstop>pushButtonCancel</tabstop>
 </tabstops>
 <resources>
  <include location="Icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButtonCancel</sender>
   <signal>clicked()</signal>
   <receiver>CreateConnection</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>311</x>
     <y>396</y>
    </hint>
    <hint type="destinationlabel">
     <x>182</x>
     <y>209</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
